### DISCUSSION TITLE
What are the differences between HoraeDB and other time-series databases?

### DISCUSSION THREAD

**Discussion by Wudadada** [CeresDB/ceresdb#1560] [Q&A] [answered]
What are its advantages? For example, compared to VictoriaMetrics?

**Comment by jiacai2050** [comment, marked as answer]
Hi, thanks for your interest.

In general, Apache HoraeDB was originally designed for analytic user case, this means data involved in one query will contains many series, just like what a AP database does.

Prometheus/VictoriaMetrics are designed for cloud-native case, APM mainly. so its query patterns are:
1. point query, to get latest status of some service.
2. range query, which may contains several series, but less than what a AP database does.


Since their goal are different, different architect are used, for HoraeDB scalability is the first goal, so all components are scalable, details can be found [here](https://horaedb.apache.org/docs/design/architecture/).
